src/public/sitemap.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+      xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
+
+<url>
+  <loc>https://portal.dev.gdi.lu/</loc>
```

Comment:
question: can we generate this automatically? These URL's should change depending on environment.

Reply:
@brunopacheco1 I did it manually once via xml-sitemaps.com for LNDC as [it was suggested by our devops](https://lnds.atlassian.net/browse/ART-8606). Then I added it for GDI and kept it open as I was not sure if it is needed or not. 

I did not go through checking if automatically it can be generated for free, but I know xml-sitemaps.com has a paid account in which we could do it.